patterns/alz/alzArm.param.json

```diff
@@ -2033,6 +2033,21 @@
         },
         "FDBackendRequestLatencyAlertState": {
           "value": "true"
+        },
```

Comment:
Please undo the modifications to the parameter file. No additional changes will be made to the Landing Zone Initiative, so no updates are necessary for the parameters until we launch a new initiative for Logic Apps.